src/resources/views/fields/date_range.blade.php

```diff
@@ -3,18 +3,22 @@
 <?php
     // if the column has been cast to Carbon or Date (using attribute casting)
     // get the value as a date string
-    if ( isset($entry) && ($entry->{$field['start_name']} instanceof \Carbon\Carbon || $entry->{$field['start_name']} instanceof \Jenssegers\Date\Date) ) {
-        $start_name = $entry->{$field['start_name']}->format( 'Y-m-d H:i:s' );
-    } else {
-        $start_name = null;
+    function formatDate($entry, $dateFieldName)
+    {
+        $formattedDate = null;
+        if (isset($entry) && !empty($entry->{$dateFieldName})) {
+            $dateField = $entry->{$dateFieldName};
+            if ($dateField instanceof \Carbon\Carbon || $dateField instanceof \Jenssegers\Date\Date) {
```

Comment:
I wonder if we should attempt to just "duck type" this (i.e. if it's an object that has the format method then call the format method).